src/frontend/next/package.json

```diff
@@ -8,7 +8,10 @@
     "start": "next start"
   },
   "dependencies": {
+    "@material-ui/core": "^4.11.2",
+    "@material-ui/icons": "^4.11.2",
     "next": "^10.0.2",
+    "prop-types": "^15.7.2",
```

Comment:
Forgot about this, are we still using prop-types? I don't see it anywhere in the code below

Reply:
I added that back into the next frontend early on, I don't believe its needed anywhere now so ill remove it.